src/features/authentication/sign-up/SignUp.controller.ts

```diff
@@ -18,4 +18,37 @@ export class SignUpController {
       return res.status(code).json(message)
     }
   }
+
+  public async updateMember(req: Request, res: Response) {
+    try {
+      const member = req.body
+      await this.signUpService.updateMember(member)
+      return res.status(200).json("Member updated!")
+    } catch (error) {
+      const { message, code } = error as HttpErrorResponse
+      return res.status(code).json(message)
```

Comment:
Derive this into separate function and use that function for all catch erros

Reply:
create a function in Response.ts
`export const SendErrorResponse = (error: HttpErrorResponse | Error, res: Response) => {
  return res.status(error instanceof HttpError ? error.code : 500).send(error.message)
}`

And use this function as below.

```suggestion
      return SendErrorResponse(error, res)
```